roachtest/inspect: use INSPECT instead of SCRUB in inspect test

spilchen opened the pull request:
Now that the INSPECT command has been implemented, we can use INSPECT instead of SCRUB to start inspect in the inspect throughput test.

Informs #154457

Epic: CRDB-30356

Release note: none

bghal reviewed (approved):
Looks good.

Not a blocker but there are also some spots in `pkg/sql/inspect/inspect_job_test.go`. (#155475 deletes them but may not be the fastest to merge.)

spilchen committed:
roachtest/inspect: use INSPECT instead of SCRUB in inspect test

Now that the INSPECT command has been implemented, we can use INSPECT
instead of SCRUB to start inspect in the inspect throughput test.

Informs #154457

Epic: CRDB-30356

Release note: none

spilchen reviewed (commented):
Seems trivial enough. I'll include it with this PR.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/155439#-:-Obe5T5ZCOWU8N3swoxX:b-nhrubg)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @spilchen)



<!-- Sent from Reviewable.io -->

spilchen commented:
TFTR!

bors r+
